service/controller/resource/organization/create.go

```diff
@@ -2,8 +2,38 @@ package organization
 
 import (
 	"context"
+	"fmt"
+	"strings"
+
+	"github.com/giantswarm/microerror"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+
+	"github.com/giantswarm/organization-operator/service/controller/key"
 )
 
 func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
+	org, err := key.ToOrganization(obj)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	for _, prefix := range forbiddenOrganizationPrefixes {
+		if strings.HasPrefix(org.ObjectMeta.Name, prefix) {
+			r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("organization name %#q cannot start with %q", org.ObjectMeta.Name, prefix))
+			return nil
+		}
+	}
+
+	orgNamespace := newOrganizationNamespace(org.ObjectMeta.Name)
+
```

Comment:
We have some logs convention for those actions and it looks like:

- `creating something`
- Either:
  - `created something`
  - `something already exists`
```suggestion

r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("creating organization namespace %#q", orgNamespace.Name))

```